packages/DApp/src/constants/peers.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * source: @see https://fleets.status.im
+ */
+
+// note!: users may experience additional latency due to cross-regional connection
+// todo: use "dynamic" discovery protocol instead
+// todo?: use a regional map together with an environment variable for the peer selection (e.g. `VERCEL_REGION`, but probably limited to  Serverless Functions)
+export const peers = {
```

Comment:
non-blocking: this should eventually be exported from `@status-im/js` and used at https://github.com/status-im/community-dapp/pull/20

Reply:
also depends on https://github.com/status-im/community-dapp/issues/23